app/markets/[id]/MarketDetails.tsx

```diff
@@ -6,9 +6,10 @@ import { getMarket } from "@/queries/omen";
 import { Button, IconButton, Tag } from "swapr-ui";
 import { remainingTime } from "@/utils/dates";
 import Link from "next/link";
+import { Address } from "viem";
 
 interface MarketDetailsProps {
-  id: string;
+  id: Address;
 }
```

Comment:
why not call id, address?

Reply:
We can call it address, but I prefer to call id as it is the market identifier which is an address.